presto-product-tests/bin/run_on_docker.sh

```diff
@@ -70,25 +70,46 @@ function stop_application_runner_containers() {
   done
 }
 
+function stop_all_containers() {
+  local ENVIRONMENT
+  for ENVIRONMENT in $(getAvailableEnvironments)
```

Comment:
Checked it :) `local` means roughly 'make changes to this variable not leak to outer scopes; inner scopes are going to see them'.